deploy/cloudformation/manifest-passthrough.yml

```diff
@@ -0,0 +1,55 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Description: >
+  - Creates a lambda to attach API Keys to Primo API queries.
+Parameters:
+  InfrastructureStackName:
+    Type: String
+    Default: mellon-app-infrastructure
+    Description: The name of the parent infrastructure/networking stack that you created. Necessary
+                 to locate and reference resources created by that stack.
+  AppConfigPath:
+    Type: String
+    Description: The path to look for the application config from
+  PassthroughUrl:
+      Type: String
+      Description: Primo passthrough url
+  PrimoApiKey:
```

Comment:
Either add `NoEcho: True` or change this to expect an SSM parameter within AppConfigPath